assignment-8-flipkart-page-1/style.css

```diff
@@ -0,0 +1,862 @@
+.mnav, .secondheader, .mspecsndetails, .mobiles {
+  display: none;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+.header1 {
+  display: flex;
+  flex-flow: row;
+  background-color: #2874F0;
+  height: 60px;
+  align-items: center;
+  padding: 0px 120px;
+  position: fixed;
+  top: 0%;
+  left: 0%;
+  z-index: 2;
+  width: 100%;
+}
+
+.navigation1 ul {
+  margin: 0px;
+  display: flex;
+  align-items: center;
+}
+.navigation1 li {
```

Comment:
be consistent. leave an empty line after each class. it is better for readability.